### PR TITLE
How-to regarding dependency mirrors added

### PR DESCRIPTION
**Pull request by bitgully**
## Summary
This adds the documentation on how to configure Paketo Buildpacks to use dependency mirrors as described in [RFC 0060](https://github.com/paketo-buildpacks/rfcs/blob/11691011512b9f075d06c31b74ea29bb5c3e5f69/text/0060-dependency-mirrors.md) 

## Checklist
* [x] I have viewed, signed, and submitted the Contributor License Agreement.
* [x] I have linked issue(s) that this PR should close using keywords or the Github UI (See [docs](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue))
* [x] I have added an integration test, if necessary.
* [x] I have reviewed the [styleguide](https://github.com/paketo-buildpacks/community/blob/main/STYLEGUIDE.md) for guidance on my code quality.
* [x] I'm happy with the commit history on this PR (I have rebased/squashed as needed).

### CODE REVIEW

**Comment by sophiewigmore** [comment]
@paketo-buildpacks/utilities-maintainers

File: content/docs/howto/configuration.md [hunk 1]
```diff
@@ -252,6 +252,73 @@ For example, to make the BellSoft Liberica JRE dependency accessible available t
    * A key/value pair where the key is equal to the `sha256` of the dependency and the value is equal to the new URI.
 4. Configure all builds with this binding.
 
+### Dependency Mirrors
+Larger (corporate) networks might have a mirror server available to cache dependencies for access from within the local network. Dependency mirrors can be used to download all (or some) buildpack dependencies from such alternative locations regardless of their versions.
```
Comment (ForestEckhardt) [inline content/docs/howto/configuration.md:256]:
```suggestion
Larger networks might have a mirror server available to cache dependencies for access from within the local network. Dependency mirrors can be used to download all (or some) buildpack dependencies from such alternative locations regardless of their versions.
```

Even if this is the state of the world most of the time I don't think that it needs to be in the documentation.

Comment (ForestEckhardt) [inline content/docs/howto/configuration.md:256]:
```suggestion
Larger (corporate) networks might have a mirror server available to cache dependencies for access from within the local network. Dependency mirrors can be used to download buildpack dependencies from such alternative locations regardless of their versions.
```

Removed to be more concise.

File: content/docs/howto/configuration.md [hunk 2]
```diff
@@ -252,6 +252,73 @@ For example, to make the BellSoft Liberica JRE dependency accessible available t
...
+
+Similarly, including the placeholder `{originalHost}` as in `https://mirror.example.org/{originalHost}` would preserve the original URI's hostname and download from `https://mirror.example.org/github.com/bell-sw/Liberica/releases/download/11.0.8+10/bellsoft-jre11.0.8+10-linux-amd64.tar.gz`.  
+
+This placeholder can also be used together with a path prefix. E.g.: `https://mirror.example.org/buildpack-dependencies/{originalHost}`
+
+#### Schemes and Authentication
+Dependency mirror URIs can either use the `https` or `file` schemes and include basic authentication credentials, should the mirror require them.  
+The credentials can be passed to the server using the format `https://[username]:[password]@mirror.example.org`.
+
+#### Setting Hostname Mirrors
```
Comment (ForestEckhardt) [inline content/docs/howto/configuration.md:296]: Could you add an example of how a URL would be transformed by each of the settings for clarity?

**Comment by ForestEckhardt** [comment]
I am not entirely sure what is wrong with the CLA but you may need to rebase your PR to remove the commit that my account is on. Sorry about that.

File: content/docs/howto/configuration.md (hunk 2 above)
Comment (bitgully) [reply content/docs/howto/configuration.md:296]: I've tried to add a little more detail on how the URL transformation takes place when hostname mirrors are used.
Please have a look and let me know if this brings more clarity.

**Review by ForestEckhardt — APPROVED** [review]

**Pull request merged**